### PR TITLE
Updated the \ README.md file to describe the \ repo.

### PR DESCRIPTION
**Pull request by tinotenda6**
Modified the README.md file

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,19 +1,88 @@
...
+            </div>
+        
+        </div>
+    
+    
     </div>
-  </body>
-</html>
+</body>
+<script>var slideIndex = 0;
```
Comment (zhaonian) [inline portfolio/src/main/webapp/index.html]: I'm not sure what the best practice is for js, please feel free to ignore this comment 😅

Would it still work if we move this into the `script.js`? It's a little hard to keep track of the variables across files.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,15 @@
...
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+window.onload = function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none"; 
+  }
+  slideIndex++;
+  if (slideIndex > slides.length) {slideIndex = 1}
```
Comment (zhaonian) [inline portfolio/src/main/webapp/script.js]: sorry I'm not very sure what these lines do. Could you update the top level documentation for this function?

will `window.onload` be called only one time when the window loads? if this is the case, isn't `slideIndex++` is always `1` ?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,124 @@
...
+  border-radius: 50%;
+  display: inline-block;
+  transition: background-color 0.6s ease;
+}
+
+.active {
+  background-color: #717171;
+}
+
+/* Fading animation */
```
Comment (zhaonian) [inline portfolio/src/main/webapp/style.css:103]: Thanks for sharing this! :)

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (tinotenda6) [reply portfolio/src/main/webapp/index.html]: Yes it would work, thanks!

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (tinotenda6) [reply portfolio/src/main/webapp/style.css:103]: Sure!

**Review by zhaonian — APPROVED** [review]

**Pull request merged**